test/test_base.py

```diff
@@ -14,6 +14,13 @@
 
 
 class TestBase(unittest.TestCase):
+
+    @contextmanager
```

Comment:
`assertIsNone` is not a context manager, according to [the docs](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertIsNone).

Reply:
Also takes another arg, `msg`.  Just checked the source to unittest2.